modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialAbility.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+import java.util.List;
+import org.eclipse.xpanse.modules.credential.enums.CredentialType;
+
+/**
+ * Credential Ability of the special plugin for the CSP.

```

Comment:
Interface to be implemented by CSP plugins to define its authentication mechanisms.